okta/resource_okta_device_assurance_policy_android_os.go

```diff
@@ -296,9 +296,9 @@ func buildDeviceAssuranceAndroidPolicyRequest(model policyDeviceAssuranceAndroid
 	android.SetName(model.Name.ValueString())
 	android.SetPlatform(okta.PLATFORM_ANDROID)
 	if len(model.DiskEncryptionType) > 0 {
-		diskEncryptionType := make([]okta.DiskEncryptionType, 0)
+		diskEncryptionType := make([]okta.DiskEncryptionTypeAndroid, 0)
```

Comment:
I can see that the need to change to the *_os.go files is from okta-sdk-golang interfaces changing. However, this should should still be noted in the PR description.

Reply:
Change in go sdk